qadence/backends/braket/backend.py

```diff
@@ -149,17 +155,27 @@ def sample(
             from qadence.transpile import invert_endianness
 
             samples = invert_endianness(samples)
-        return samples
+        if error is not None:
+            error_fn = error.get_error_fn()
+            return error_fn(counters=samples, n_qubits=circuit.abstract.n_qubits)  # type: ignore
+        else:
+            return samples
 
     def expectation(
         self,
         circuit: ConvertedCircuit,
         observable: list[ConvertedObservable] | ConvertedObservable,
         param_values: dict[str, Tensor] = {},
         state: Tensor | None = None,
-        protocol: Measurements | None = None,
+        measurement: Measurements | None = None,
+        error: Errors | None = None,
         endianness: Endianness = Endianness.BIG,
     ) -> Tensor:
+        if error is not None:
+            logger.warning(
+                f"Errors of type {error} are not implemented for expectation yet. "
+                "This is ignored for now."
+            )
```

Comment:
Exactly. Thing is, I thought it would make sense to at least inform the user in case he/she falls here instead of carrying on and silently ignoring the error. As a matter of fact, it should also hold for other protocols IMHO. About resolving it, it is the sense of the experiment composer thing that we discussed.

Reply:
Yes, makes sense.

Reply:
Done.